ai21_tokenizer/utils.py

```diff
@@ -1,6 +1,9 @@
+import asyncio
```

Comment:
Can you rename the file to `file_utils.py`?

Reply:
Done.